terraform/destroy.sh

```diff
@@ -28,6 +28,16 @@ acct=$(gcloud info --format="value(config.account)")
 SCRIPT_DIR=$(realpath $(dirname "$0"))
 cd $SCRIPT_DIR
 
+# For the purposes of telemetry
+# session is defined as the current time the user runs destroy.sh file
+SESSION=$(python3 -c "import uuid; print(uuid.uuid4())")
+
+# this function sends de-identified information to the Google Cloud Platform database
+# on what events occur in users' Sandbox projects for development purposes
+sendTelemetry() {
+  python3 telemetry.py --session=$SESSION --project_id=$1 --event=$2 --version=$VERSION
+}
```

Comment:
It seems a little strange to re-define telemetry.py as sendTelemetry. Why not just rename the file to sendTelemetry if you prefer this?

Reply:
I don't necessarily mean to re-define telemetry.py here: I'm actually trying to reduce redundancies with sending the same `SESSION` and `VERSION` variables in every instance telemetry is sent.